src/provider/HOCs/withVariableContext.js

```diff
@@ -2,6 +2,16 @@ import { getVariablesForElement } from '@bpmn-io/extract-process-variables/zeebe
 import { useEffect, useState } from '@bpmn-io/properties-panel/preact/hooks';
 import { useService } from '../../hooks';
 
+function useServiceIfAvailable(service, fallback) {
+  const resolved = useService(service, false);
+
+  if (!resolved) {
+    return fallback;
+  }
+
+  return resolved;
+}
+
```

Comment:
using the `variableResolver` should be optional, so we check if the plugin was added. If it is not available, we just use `extract-process-variables` as before